ckanext/canada/cli.py

```diff
@@ -1873,3 +1873,30 @@ def iter_records():
         outf.close()
     if verbose:
         click.echo("Done!")
+
+
+@canada.command(short_help="Deletes old database triggers.")
+@click.option('-v', '--verbose', is_flag=True, type=click.BOOL, help='Increase verbosity.')
+def delete_old_triggers(verbose=False):
+    """
+    Delete old, unused database triggers.
+    """
+    _drop_function('no_surrounding_whitespace_error', verbose)
+    _drop_function('year_optional_month_day_error', verbose)
+    _drop_function('choices_from', verbose)
+    _drop_function('integer_or_na_nd_error', verbose)
+    _drop_function('inventory_trigger', verbose)
```

Comment:
https://docs.ckan.org/en/2.10/maintaining/datastore.html#ckanext.datastore.logic.action.datastore_function_delete

Reply:
@wardi because of our addition to the argmode and extra args stuff, the `datastore_delete` CKAN action did not work for me. So just did it in this crappy way for now